### PR TITLE
fix: process URLs with no query string

### PR DESCRIPTION
**Pull request by benmccann**
- **Quick Checklist**

* [X] I have read [the contributing guidelines](../CONTRIBUTING.md)
* [X] I have written new tests, as applicable (for bug fixes / features)
* [X] Docs have been added / updated (for bug fixes / features)
* [X] I have added a changeset, if applicable

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Fixes https://github.com/JonasKruckenberg/imagetools/issues/393

- **What is the new behavior (if this is a feature change)?**

Processes URLs with no query string

- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this
  PR?)

While this changes the behavior, I'd call this a bug fix rather than a breaking change

- **Other information**:

### CODE REVIEW

File: packages/rollup/README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ ___
 
 Which paths to include when processing images.
 
-**`default`** '**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}?*'
+**`default`** ['**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}', '**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}?*']
```
Comment (JonasKruckenberg) [inline packages/rollup/README.md:103]: This renders weirdly in markdown, maybe you need to somehow escape the asterisks?

File: packages/vite/README.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ ___
 
 Which paths to include when processing images.
 
-**`default`** '**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}?*'
+**`default`** ['**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}', '**\/*.{heic,heif,avif,jpeg,jpg,png,tiff,webp,gif}?*']
```
Comment (JonasKruckenberg) [inline packages/vite/README.md:107]: Same here

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]

File: packages/rollup/README.md (hunk 1 above)
Comment (benmccann) [reply packages/rollup/README.md]: fixed. escaping the asterisks worked, but I decided to put it in backticks instead, which also worked and looked a bit better to me

File: packages/vite/README.md (hunk 2 above)
Comment (benmccann) [reply packages/vite/README.md]: fixed

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]
Actually, if you could quickly change the tests that test urls without directives to ensure images are generated that'd be the last thing 👍

**Comment by benmccann** [comment]
Ok, I've updated a couple of tests to ensure it works with no query string

**Comment by JonasKruckenberg** [comment]
Alright! Thanks 😁

**Pull request merged**